tests/test_time_series_utils.py

```diff
@@ -32,13 +32,13 @@ def test_get_variable_name_and_lag(self):
 
         # With special characters
         self.assertTupleEqual(
-            get_variable_name_and_lag('1@£$%^&*()-+=*/_\\"\'<>?*~.'), ('1@£$%^&*()-+=*/_\\"\'<>?*~.', 0)
+            get_variable_name_and_lag('1@£$%^&*()-+=*/\n_\\"\'<>?*~.'), ('1@£$%^&*()-+=*/\n_\\"\'<>?*~.', 0)
```

Comment:
I am not fluent in regex, but does the change in this test mean that this change is not backwards compat? i.e. if you don't change this line, does that test pass? If so, I would leave existing tests as is and add more tests for new cases.

Reply:
No the test works fine without modificatoin, I was just including `\n` in the list of special characters. I can have it separate though.